### PR TITLE
ci: switch to blacksmith runners

### PR DESCRIPTION
**Pull request by milapsheth**

### CODE REVIEW

File: .github/workflows/codecov.yaml [hunk 1]
```diff
@@ -13,22 +13,16 @@ concurrency:
 
 jobs:
   coverage:
-    name: Code Coverage
```
Comment (re1ro) [inline .github/workflows/codecov.yaml:16]: Keep the name?

File: .github/workflows/lint.yaml [hunk 2]
```diff
@@ -9,20 +9,16 @@ concurrency:
 
 jobs:
   lint:
-    strategy:
```
Comment (re1ro) [inline .github/workflows/lint.yaml:12]: Add name?

File: .github/workflows/slither.yaml [hunk 3]
```diff
@@ -8,21 +8,17 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  analyze:
```
Comment (re1ro) [inline .github/workflows/slither.yaml:11]: Name?

File: .github/workflows/test.yaml [hunk 4]
```diff
@@ -9,20 +9,16 @@ concurrency:
 
 jobs:
   test:
-    strategy:
```
Comment (re1ro) [inline .github/workflows/test.yaml:12]: Name?

File: .github/workflows/codecov.yaml (hunk 1 above)
Comment (milapsheth) [reply .github/workflows/codecov.yaml:16]: it already shows the name of the workflow from the top of the file

**Review by re1ro — APPROVED** [review]

**Pull request merged**